### ISSUE TITLE
Steps to help user when raw.githubusercontent.com is blocked (It's blocked by certain network providers in India)

### ISSUE THREAD

**Issue by pavish** [centerofci/mathesar#2593]
## Description
* raw.githubusercontent.com is geo-blocked by certain network providers in India. Requests to the domain get timed out.
* We use this content domain in our quick start command and also inside install.sh to download the docker-compose.yml file.

## Steps to take
1. We should update our quickstart command to be able to throw some error or warning if the request fails. Currently, it exits without printing anything when an error occurs, and the user has no idea what's going on.
1. We should update our docs or have an open issue (maybe this one), so that users facing this problem will try any workarounds posted. The domain is geo-blocked, some providers block it differently from others by just blocking it at the DNS level. Using a VPN should work for all cases.
1. We should try to host these 2 files separately in our own domain or with a different content provider, if possible. - tracked in #2613 

## Related
* [Mathesar-developers mail thread](https://groups.google.com/a/mathesar.org/g/mathesar-developers/c/A0GupMHPRow/m/oA7l9INqAQAJ)

### Script download feedback from one of our usability testers
> Also I was going through more installations(recently installing helm for kubernetes testing) and it follows the trend of;
> 1) Copying the script
> 2) Making script executable
> 3) Running the .sh script file.
> 
> I wanted to say that this industry standard is a bit better than currently wget for the reason of that if we want to check the script, or modify it, or re run it then we have the file locally.
> 
> By copying I mean download with wget.

**Comment by pavish** [comment]
@kgodey I notice a good deal of traffic from India, and I think we should try to do something about this for the release. I've placed it in 'Launch nice-to-haves', we might want it to be part of the blockers.

**Comment by pavish** [comment]
@mathemancer 

#2609 Seems to handle the failure cases within install.sh

The quickstart command needs to be updated in the docs and #2601 doesn't seem to include the curl flag for failure.

It's still:
`bash <(curl -sL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)`

I believe it should be enough to update this to:
`bash <(curl -sfL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)`

**Comment by pavish** [comment]
@mathemancer Can we update the quick start command to this:

```
bash <(curl -sSL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
```

The arg `S` seems to print failures, even when installing in silent mode.

```
pavish@Pavishs-MacBook-Pro mathesar % bash <(curl -sSL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
curl: (7) Failed to connect to raw.githubusercontent.com port 443: Operation timed out
```

**Comment by mathemancer** [comment]
> @mathemancer Can we update the quick start command to this:
> 
> ```
> bash <(curl -sSL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
> ```
> 
> The arg `S` seems to print failures, even when installing in silent mode.
> 
> ```
> pavish@Pavishs-MacBook-Pro mathesar % bash <(curl -sSL https://raw.githubusercontent.com/centerofci/mathesar/master/install.sh)
> curl: (7) Failed to connect to raw.githubusercontent.com port 443: Operation timed out
> ```

Yes, I think that works.

**Comment by kgodey** [comment]
I'll close this issue in #2612. I've created #2613 to track further work. I think our failure message is clear enough for now.

**Comment by rajatvijay** [comment]
This is a prominent issue. I am also facing it. Using VPN solves it though.
